### PR TITLE
Show standards on lesson plan page

### PR DESCRIPTION
**Pull request by davidsbailey**
Starts [PLAT-330]. See [spec](https://docs.google.com/document/d/1QFzo3vdeDyqiuOyVWhmHesuSNOevNhvwSRK9kVyblxQ/edit#heading=h.vighmxgsfwp3) -> "Teacher-Facing Requirements" for how it is supposed to look. This gets standards showing in the right structure on the lesson overview page. I thought this was a good stopping point, so I left a few details to be taken care of in follow up work below.

A few interesting implementation details:
1. I'm using summary/details to handle the expand/collapse. this makes the components stateless.
2. because of (1), I am able to use PureComponent, which tells react not to re-render if the props do not change. 
3. I'm using multiple classes in one .jsx file, with only one of them being exported, since this seemed like a good way to encapsulate the "private" classes. not sure if this is controversial or not?

Please nit! My React style may be a bit rusty.

## screenshots

the standards start off collapsed. this screenshot shows what they look like when a few toggles are expanded:

Screen Shot 2021-03-18 at 4 38 00 PM

video showing expand/collapse:

https://user-images.githubusercontent.com/8001765/111711407-da637880-8808-11eb-9bb6-6c65de6cb90d.mov

## Follow-up work
* make styling look more like it does in the specs
* make framework and categories expanded by default for the first standard
* make standards appear correctly in print view (TBD, see comments in [spec](https://docs.google.com/document/d/1QFzo3vdeDyqiuOyVWhmHesuSNOevNhvwSRK9kVyblxQ/edit))

[PLAT-330]: https://codedotorg.atlassian.net/browse/PLAT-330

### CODE REVIEW

File: apps/src/templates/lessonOverview/LessonStandards.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          <summary>
+            {shortcode}
+            {' - '}
+            {description}
+          </summary>
+          <ul style={{listStyleType: 'none'}}>
+            {Object.keys(standardsByCategory).map(categoryShortcode => {
+              const standards = standardsByCategory[categoryShortcode];
+              return (
+                <Category
```
Comment (bethanyaconnor) [inline apps/src/templates/lessonOverview/LessonStandards.jsx:90]: Is it ever possible for a category to have multiple parents?

File: apps/test/unit/templates/lessonOverview/LessonOverviewTest.js [hunk 2]
```diff
@@ -176,4 +179,28 @@ describe('LessonOverview', () => {
...
+    ];
+    const lesson = {
+      ...defaultProps.lesson,
+      standards: standards
+    };
+    const wrapper = shallow(
+      <LessonOverview {...defaultProps} lesson={lesson} />
+    );
+
+    expect(wrapper.containsMatchingElement(<h2>Standards</h2>)).to.be.true;
```
Comment (bethanyaconnor) [inline apps/test/unit/templates/lessonOverview/LessonOverviewTest.js:204]: Is it worth testing the contents of the standards section, either in this test or in a separate test file for `LessonStandards`?

Comment (davidsbailey) [reply apps/test/unit/templates/lessonOverview/LessonOverviewTest.js:204]: yes, apologies... I will add LessonStandardsTest.js and a .story.jsx.

File: apps/src/templates/lessonOverview/LessonStandards.jsx (hunk 1 above)
Comment (davidsbailey) [reply apps/src/templates/lessonOverview/LessonStandards.jsx:90]: good question. not currently. some frameworks have multiple hierarchies along which they categorize their standards, but at present we only plan to provide one hierarchy for each framework. This info on NGSS standards may give an idea for some potential future needs in this space, though: https://www.nextgenscience.org/understanding-standards/understanding-standards

**Review by dmcavoy — COMMENTED** [review]
I don't have anything to add beyond what Bethany already commented on.

File: apps/test/unit/templates/lessonOverview/LessonOverviewTest.js (hunk 2 above)
Comment (davidsbailey) [reply apps/test/unit/templates/lessonOverview/LessonOverviewTest.js:204]: @bethanyaconnor please take another look!

**Review by bethanyaconnor — APPROVED** [review]
Looks great! Thanks for adding all the tests!

**Pull request merged**